### PR TITLE
This hides the government row from the list of Fuel Suppliers

### PR DESCRIPTION
**Pull request by amichard**
#231 

This hides the Government row from the Fuel Suppliers Table

### CODE REVIEW

File: backend/api/viewsets/Organization.py [hunk 1]
```diff
@@ -36,6 +36,14 @@ def get_serializer_class(self):
         else:
             return self.serializer_classes['default']
 
+    def list(self, request):
+        fuel_suppliers = Organization.objects.filter(
+            type=OrganizationType.objects.get(type="Part3FuelSupplier")) \
```
Comment (Sybrand) [inline backend/api/viewsets/Organization.py]: Could you add a comment here to explain why filtering on Part3FuelSupplier is being done?

Comment (amichard) [reply backend/api/viewsets/Organization.py]: Sure. Updated.

**Comment by justinlepitzki** [comment]
@kuanfandevops yes please deploy to test. Thank you!

**Comment by kuanfandevops** [comment]
@justinlepitzki @matthewhall78 deployed on test. Let me know when you want it with previous changes deployed on prod.

**Pull request merged**